File: c_practice_file/factorial.c
////////// factorial //////////

#include<stdio.h>
int main()
{
int i=1,x,fact=1;
printf("enter the number ");
scanf("%d",&x);
while(i<=x)
{
fact=fact*i;
i++;
}
printf("fact %d \n",fact);
/*

for(i=1;i<=x;i++)
{
fact = fact*i;
}
printf("%d fact = %d\n",x,fact);

}*/
}

File: python_lab/for_loop/function/factorial.py
'''
3. create a function to calculate and
return factorial of a number passed in
argument
'''
print("3. create a function to calculate and return factorial of a number passed in argument")
def fact (n):
    f=1
    while n!=0:
        f=f*n
        print("f - ",f)
        n-=1
    return f

n=int(input("enter last Number :- "))
print("factorial - ",fact(n))